### PR TITLE
Business dashboard theming for ListItemAction & ListItemSection

### PR DESCRIPTION
**Pull request by dimafeldman**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6228

File: src/PopoverMenu/PopoverMenu.js [hunk 1]
```diff
@@ -297,28 +298,32 @@ class PopoverMenu extends React.PureComponent {
...
-        tabIndex={-1}
-        moveBy={moveBy}
-        maxHeight={maxHeight}
-        zIndex={zIndex}
-      >
-        {({ toggle, open, close }) =>
-          this._renderTriggerElement({ toggle, open, close })
-        }
-      </DropdownBase>
+      <ThemeProviderConsumerBackwardCompatible>
```
Comment (shlomitc) [inline src/PopoverMenu/PopoverMenu.js:301]: this solution should be applied in the `Popover` component instead

File: src/Themes/businessDashboard/docs/components/ListItemAction.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+export const playground = `
+  <ThemeProvider theme={theme()}>
+    <div
+      style={{
+        width: '284px',
+        height: '240px',
+      }}
+    >
```
Comment (shlomitc) [inline src/Themes/businessDashboard/docs/components/ListItemAction.js:8]: remove

File: src/Themes/businessDashboard/docs/components/ListItemAction.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      <DropdownLayout
+        visible
```
Comment (shlomitc) [inline src/Themes/businessDashboard/docs/components/ListItemAction.js:10]: add `inContainer`

File: src/Themes/businessDashboard/components/DropdownLayout.st.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
+:import {
+  -st-from: "../../../DropdownLayout/DropdownLayout.st.css";
+  -st-default: DropdownLayout;
+}
+
+:import {
+  -st-from: '../../../Foundation/stylable/default-scroll-bar.st.css';
+  -st-named: defaultScrollBar;
+}
```
Comment (shlomitc) [inline src/Themes/businessDashboard/components/DropdownLayout.st.css:9]: redundant

**Review by shlomitc — CHANGES_REQUESTED** [review]

File: src/Themes/businessDashboard/BusinessDashboard.st.css [hunk 5]
```diff
@@ -210,3 +259,9 @@
     -st-mixin: SidebarSectionItemOverrides();
   }
 }
+
+@st-scope * {
+  PopoverMenu.root {
```
Comment (shlomitc) [inline src/Themes/businessDashboard/BusinessDashboard.st.css:264]: I think that once this is done in the `Popover` component, then change it to `Popover`

File: src/Themes/businessDashboard/components/DropdownLayout.st.css [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+:import {
+  -st-from: '../foundations/colors.st.css';
+  -st-named: D40, D55;
+}
+
+.root {
+  -st-extends: DropdownLayout;
+}
+
+.root::options::-webkit-scrollbar-thumb {
```
Comment (shlomitc) [inline src/Themes/businessDashboard/components/DropdownLayout.st.css:20]: I think you need to control also the `::-webkit-scrollbar`. it seems like the width is too big.
image

Comment (dimafeldman) [reply src/Themes/businessDashboard/components/DropdownLayout.st.css:20]: fixed

**Pull request merged**